api/src/test/java/uk/gov/dhsc/htbhf/card/service/CardServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.dhsc.htbhf.card.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.dhsc.htbhf.card.model.DepositFundsRequestDTO;
+import uk.gov.dhsc.htbhf.card.model.DepositFundsResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static uk.gov.dhsc.htbhf.card.testhelper.DepositFundsRequestDTOTestDataFactory.aValidDepositFundsRequest;
+import static uk.gov.dhsc.htbhf.card.testhelper.DepositFundsResponseTestDataFactory.aValidDepositFundsResponse;
+import static uk.gov.dhsc.htbhf.card.testhelper.TestConstants.DEPOSIT_REFERENCE_ID;
+
+@ExtendWith(MockitoExtension.class)
+class CardServiceTest {
+
+    private static final String BASE_URI = "http://localhost:8120";
+
+    @Mock
+    private RestTemplate restTemplate;
+
+    @InjectMocks
```

Comment:
Don't need the inject mocks if we're newing it in the init method.

Reply:
Yes, good spot, I'll remove.